XbSymbolDatabase.c

```diff
@@ -1383,14 +1387,17 @@ bool XbLibraryScan(custom_scan_func_t custom_scan_func,
 
                         scan_ret = custom_scan_func(LibraryFlag, pSectionScan, BuildVersion, LibraryStr, register_func, xb_start_virt_addr);
 
-                        if (!scan_ret) {
-                            continue;
+                        if (scan_ret) {
+                            // let's return true here instead of waste the loops for nothing.
+                            return scan_ret;
                         }
-                        return scan_ret;
                     }
                 }
             }
-            break;
+            // Use the break if there are 2+ bit flags set such as include LTCG flag in std flag's oovpa database like D3D8.
+            if ((SymbolDBList[d2].LibSec.library & ~LibraryFlag) == 0) {
```

Comment:
However, if we do this way... and the database aren't in order such as LTCG database could be at the top and the std database below it. It will only stop at the LTCG database.